### DISCUSSION TITLE
the rendered logo size is too small, how to adjust it.

### DISCUSSION THREAD

**Discussion by jackywu** [web-infra-dev/rspress#994] [Q&A]
how to adjust the logo image size ?
as you see, my logo image has good size and resolution.

![Xnip2024-04-20_22-58-18](https://github.com/web-infra-dev/rspress/assets/587955/26a80c4b-95dd-452b-912e-fe8f08ce3341)

![Xnip2024-04-20_23-00-45](https://github.com/web-infra-dev/rspress/assets/587955/d8df32f6-7d0a-4b20-8e74-f6453abc6ef5)

**Comment by Timeless0911** [comment]
You can use [globalStyles](https://rspress.dev/plugin/system/plugin-api#globalstyles) to modify some css styles of `.rspress-logo`
